pygorithm/math/factorial.py

```diff
@@ -1,34 +1,10 @@
 """
```

Comment:
Delete this file, as there is a reason why I didn't use builtin factorial from`math` module. Main purpose behind this module is to teach people how to implement various DS and Algos in Python. Hence, we don't need this file as this is already implemented

Reply:
Done